youtube-client-app/src/app/auth/pages/login/login.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+import { AuthService } from '../../services/auth.service';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss'],
+})
+export class LoginComponent implements OnInit {
+
+  constructor(public auth: AuthService) { }
```

Comment:
it usually uses private auth: AuthService and add public methods in class for login/logout